Playdown.swift

```diff
@@ -179,10 +181,25 @@ struct Playdown {
                 }
                 
             } catch {
-                print(error)
+                
+                //Handle case for single playground page or single swift file
+                var streamPath = filename
+                let playgroundName = playgroundPath.replacingOccurrences(of: playgroundDirectory + "/", with: "")
+                
+                if let playPathURL = URL(string: filename) {
```

Comment:
combine the next if as a ```where```  clause. also make it playPathURL.pathExtension != "swift" instead